### PR TITLE
Upgrade to Helm 3

### PR DESCRIPTION
**Pull request by noamt**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,8 +15,7 @@ VOLUME /data
 
 USER gkh
 
-RUN helm init --client-only && \
-    helm plugin install https://github.com/futuresimple/helm-secrets.git || \
+RUN helm plugin install https://github.com/futuresimple/helm-secrets.git || \
```
Comment (monotek) [inline Dockerfile]: could you please add the changes requested in: https://github.com/kiwigrid/gcloud-kubectl-helm/commit/cba4a65e284ef680aaba79d3d54f94f765b6bfc4

Comment (noamt) [reply Dockerfile]: Yes

**Comment by noamt** [comment]
So I've corrected the helm secrets plugin expression but it looks like it's hitting an issue with locating SOPS.
It looks like the issue is recurring. It's mentioned [here](https://github.com/futuresimple/helm-secrets/issues/34).
Perhaps this shouldn't be part of the Dockerfile, and be manually installed by users who need this?

**Comment by monotek** [comment]
I've fixed the sops problem. 
The version tag changed from 3.4.0 to v3.5.0 :/

**Pull request merged**